app/src/main/res/values/strings.xml

```diff
@@ -105,5 +105,18 @@
     <string name="association_details_events">Events</string>
     <string name="list_drawer_leave_event">Leave Event</string>
     <string name="event_successfully_left">Event successfully left!</string>
+    <string name="help_screen_how_can_i_find_events">How can I find events?</string>
+    <string name="help_screen_how_can_i_find_events_description">Good question! By default you see all events that match your interests. You can filter by only one of your interests by tapping on the chips on the top of the map.\n\nTap on the search icon in the bottom left part of the screen to open the search options. This mode will consider all events, not only your interests. Perfect for discovering new events on the campus! In the top right corner you have an icon button to cancel the search.</string>
+    <string name="help_screen_create_event">I want to create an event</string>
+    <string name="help_screen_create_event_description">Great! You have to options: either you open the hamburger menu and tap on \'Create Event\' or you simply long press on the map. Then fill in the details and save it! It\'s this simple.</string>
```

Comment:
small typo : "You have two options" instead of "You have to options"
small detail : "It's that simple" instead of "it's this simple" is more common